### PR TITLE
Update experiment-management-api-experiments.md

### PR DESCRIPTION
**Pull request by akhil-prakash**

### CODE REVIEW

File: content/collections/experiment-apis/en/experiment-management-api-experiments.md [hunk 1]
```diff
@@ -1027,8 +1027,8 @@ Edit an experiment.
 
 |<div class="med-big-column">Name</div>|Requirement|Type|Description|
 |---|---|---|---|
-|`group_type`| Optional | string | Group type of the filter; can be null. Can be `User` value or one of the group values, eg `org _id`, `org name` |
-|`subprop_key`| Required | string | Filter's key; can be null. |
+|`group_type`| Optional | string | Group type of the filter; can be null. Can be `User` value or one of the group values, eg `org_id`, `org name` |
+|`subprop_key`| Required | string | Filter's key; cannot be null. |
```
Comment (akhil-prakash) [inline content/collections/experiment-apis/en/experiment-management-api-experiments.md:1031]: was this documented that the key can file null somewhere?

File: content/collections/experiment-apis/en/experiment-management-api-experiments.md [hunk 2]
```diff
@@ -1027,8 +1027,8 @@ Edit an experiment.
 
 |<div class="med-big-column">Name</div>|Requirement|Type|Description|
 |---|---|---|---|
-|`group_type`| Optional | string | Group type of the filter; can be null. Can be `User` value or one of the group values, eg `org _id`, `org name` |
-|`subprop_key`| Required | string | Filter's key; can be null. |
+|`group_type`| Optional | string | Group type of the filter; can be null. Can be `User` value or one of the group values, eg `org_id`, `org name` |
```
Comment (markzegarelli) [inline content/collections/experiment-apis/en/experiment-management-api-experiments.md:1030]:
```suggestion
|`group_type`| Optional | string | Group type of the filter; can be null. Can be `User` value or one of the group values, for example `org_id` or `org name` |
```

File: content/collections/experiment-apis/en/experiment-management-api-experiments.md (hunk 1 above)
Comment (markzegarelli) [inline content/collections/experiment-apis/en/experiment-management-api-experiments.md:1031]:
```suggestion
|`subprop_key`| Required | string | Filter's key; can't be null. |
```

Comment (markzegarelli) [reply content/collections/experiment-apis/en/experiment-management-api-experiments.md:1031]: I'll need to check. I don't remember documenting this as such.

**Review by markzegarelli — CHANGES_REQUESTED** [review]

**Review by markzegarelli — APPROVED** [review]

**Pull request merged**